qcodes/tests/test_station.py

```diff
@@ -646,3 +646,19 @@ def test_deprecated_type_keyword():
     with warnings.catch_warnings(record=True) as w:
         st.load_instrument('mock')
     assert issubclass(w[-1].category, QCoDeSDeprecationWarning)
+
+
+def test_deprecated_limits_keyword_as_string():
+    st = station_from_config_str("""
+instruments:
+  mock:
+    driver: qcodes.tests.instrument_mocks.DummyInstrument
+    init:
+      gates: {"ch1"}
+    parameters:
+      ch1:
+        limits: -10, 10
+    """)
+    with warnings.catch_warnings(record=True) as w:
+        st.load_instrument('mock')
+        assert issubclass(w[-1].category, QCoDeSDeprecationWarning)
```

Comment:
newline and indentation :)
```suggestion
    assert issubclass(w[-1].category, QCoDeSDeprecationWarning)

```